src/components/atoms/CodeEditor.tsx

```diff
@@ -122,6 +124,15 @@ class CodeEditor extends React.Component<CodeEditorProps> {
     }
   }
 
+  handleCodeMirrorPaste = (
+    _editor: CodeMirror.Editor,
+    event: ClipboardEvent
+  ) => {
+    if (this.props.onPaste != null) {
```

Comment:
I think we should change this so it cover `onPaste` being `underfined` too
```suggestion
    if (this.props.onPaste) {
```

Reply:
![image](https://user-images.githubusercontent.com/30511303/82892441-bc621a00-9f4f-11ea-9466-30875cfdb43d.png)
I don't really like to write `!= null`, I would have put that instead (but I'm following how it was done before) : `if (!this.props.onPaste) return`

Reply:
ah that's true, damn it! JS confused me every single time!